uiServices/test/createConfigFile_test.go

```diff
@@ -0,0 +1,202 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+	"github.com/DATA-DOG/godog"
+	"github.com/DATA-DOG/godog/gherkin"
+	"io/ioutil"
+	"net/http"
+	"io"
+	"strings"
+	"os"
+	"github.com/Sirupsen/logrus"
+	"github.com/xtracdev/automated-perf-test/uiServices/src"
+	"encoding/xml"
+	"github.com/xtracdev/automated-perf-test/perfTestUtils"
+	"encoding/json"
+	"reflect"
+)
+
+type apiFeature struct {
+	resp   *http.Response
```

Comment:
add a `requestBody string` in the struct that you can use to store and read the request payload